### DISCUSSION TITLE
Renovate GitHub auto-merging stuck

### DISCUSSION THREAD

**Discussion by Jamesking56** [renovatebot/renovate#12291] [Request Help]
I've enabled Renovate's auto-merging to use GitHub's native auto-merge button but it appears to get stuck:

![Screenshot 2021-10-23 at 8 28 42 pm](https://user-images.githubusercontent.com/253237/138569033-c0a02b4a-ab4b-450e-802e-848ee21406e6.png)

Any idea why this gets stuck on 'Attempting to auto-merge...'?

**Comment by rarkins** [comment]
That looks like a GitHub bug.

Find the job log which matches the PR creation time to see what the logs say

**Reply by Jamesking56** [reply]
Yeah opened a ticket with GitHub as I think its them, Renovate appears to have set the flag in the API fine.
